src/libs/Localize/index.ts

```diff
@@ -225,5 +226,19 @@ function getDevicePreferredLocale(): Locale {
     return RNLocalize.findBestAvailableLanguage([CONST.LOCALES.EN, CONST.LOCALES.ES])?.languageTag ?? CONST.LOCALES.DEFAULT;
 }
 
-export {translate, translateLocal, formatList, formatMessageElementList, getDevicePreferredLocale};
+/**
+ * This function match the message in the translation mapping and return the translation.
```

Comment:
For a given backend error message, swap it out for an equivalent error message that is localized.